src/Sylius/Behat/Context/Ui/Admin/ManagingCountriesContext.php

```diff
@@ -138,13 +164,13 @@ public function iShouldBeNotifiedAboutSuccessfulEdition()
     }
 
     /**
-     * @Then /^(country "[^"]+") should appear in the store$/
+     * @Then /^the (country "([^"]+)") should appear in the store$/
```

Comment:
Added parentheses should be removed. Eg.
```php
/**
 * @Then /^(country "([^"]+)") should have description "([^"]+)"$/
 */
public function krzysztofKrawczyk(CountryInterface $country, $description) {}
```

Given the step `country "Poland" should have description "Stronk"` will match `Poland` as `$description` instead of `Stronk` (it will be used the third, unexisting parameter).